.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
```

Comment:
You will want to remove these vscode files from the branch.  you can do this with 'git rm' or update your .git-ignore file to ignore them.

same with the aerabati_cirq_fork files.